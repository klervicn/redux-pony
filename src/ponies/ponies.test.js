import React from 'react';
import Ponies from './ponies';
import renderer from 'react-test-renderer';

it('renders correctly', () => {
  const tree = renderer
    .create(
      <Ponies
        poniesList={[
          { id: 15, name: 'Petit Poney' },
          { id: 20, name: 'Mycroft' }
        ]}
        onUpdateClick={jest.fn}
        onDeleteClick={jest.fn}
      />
    )
    .toJSON();

  expect(tree).toMatchSnapshot();
});
