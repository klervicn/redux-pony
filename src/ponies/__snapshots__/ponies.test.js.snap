// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<ul>
  <li
    style={
      Object {
        "listStyleType": "none",
      }
    }
  >
    <input
      onChange={[Function]}
      type="text"
      value="Petit Poney"
    />
    <button
      onClick={[Function]}
    >
      Update
    </button>
    <button
      onClick={[Function]}
    >
      Delete
    </button>
  </li>
  <li
    style={
      Object {
        "listStyleType": "none",
      }
    }
  >
    <input
      onChange={[Function]}
      type="text"
      value="Mycroft"
    />
    <button
      onClick={[Function]}
    >
      Update
    </button>
    <button
      onClick={[Function]}
    >
      Delete
    </button>
  </li>
</ul>
`;
